app/views/shopping_lists/index.html.erb

```diff
@@ -0,0 +1,43 @@
+
+<main>
+        <section class="shopping_list1">
+            <header>
+                <h2>Shopping List</h2>
+            </header>
+        </section>
+
+        <section class="shopping_list2">
+
+            <article>
+                <div>
+                    <p>Amount of food items to buy: <%= @shopping_lists.length%> </p>
+                </div>
+                <div>
+                    <p>Total value of food needed: $<%=@total_price%></p>
+                </div>
+            </article>
+
+            <table class="styled_table">
+
+                <thead>
+                    <tr class="tr">
+                        <th>Food</th>
+                        <th>Quantity</th>
+                        <th>Price</th>
+                    </tr>
+                </thead>
+                <tbody> 
+                  <% @shopping_lists.each do |food| %>
+                        <tr class="active_row custom_row1"> 
+                            <th class="custom_font1"><%= food[:name]%></th>
+                            <th class="custom_font1"><%= food[:quantity]%></th>
+                            <th class="custom_font1"><%= food[:price]%></th>
+                        </tr>
+                    <% end %>
+                </tbody>
+
+            </table>
+
+
+        </section>
+    </main>
```

Comment:
- [ ] It will be great to ensure that the shopping list index page matches the given wireframe. This is critical for your project to be approved. 

This is how your page looks now ⬇️ 
![Screenshot 2023-07-14 at 23 07 38](https://github.com/ghreza-crypto/Recipe-app/assets/32361963/f9af8d45-3153-4849-9a50-5df60df6efec)

This is how it should look ⬇️ 
![Screenshot 2023-07-14 at 23 08 17](https://github.com/ghreza-crypto/Recipe-app/assets/32361963/2f25d0f1-76ad-45ee-9492-15f484bf5e35)